src/integration/pages/homePage/canonicalTests.js

```diff
@@ -0,0 +1,19 @@
+import { runCoreCanonicalTests } from '../../common';
+import runCrossPlatformTests from './crossPlatformTests';
+
+export default service => {
+  runCrossPlatformTests(service);
+  runCoreCanonicalTests();
+  describe('hierarchical grid', () => {
+    const { pageData } = window.SIMORGH_DATA;
+    const numberOfItems = pageData.curations.length;
+    const hierarchicalGrid = document.querySelector(
+      '[data-testid="hierarchical-grid"]',
+    );
+    const numberOfcurations = document.querySelectorAll('h2').length;
+    it('is displayed', () => {
+      expect(hierarchicalGrid).toBeInTheDocument();
+      expect(numberOfcurations).toEqual(numberOfItems);
+    });
```

Comment:
Can we move this to `crossPlatformTests` or perhaps `pageContentTests`, or does it not work on amp?

Reply:
Yeah, this does not work on amp.